addons/project/views/project_views.xml

```diff
@@ -105,30 +106,34 @@
-                                of the default labels.
+                                At each stage, employees can block tasks or mark them as ready for the next step.
+                                You can customize here the labels for each state.
                             </p>
-                            <label for="legend_normal" string=" " class="o_status oe_project_kanban_legend"
-                                title="Task in progress. Click to block or set as done."
-                                aria-label="Task in progress. Click to block or set as done." role="img"/>
-                            <field name="legend_normal" nolabel="1"/>
-                            <label for="legend_blocked" string=" " class="o_status o_status_red oe_project_kanban_legend"
-                                title="Task is blocked. Click to unblock or set as done."
-                                aria-label="Task is blocked. Click to unblock or set as done." role="img"/>
-                            <field name="legend_blocked" nolabel="1"/>
-                            <label for="legend_done" string=" " class="o_status o_status_green oe_project_kanban_legend"
-                                title="This step is done. Click to block or set in progress."
-                                aria-label="This step is done. Click to block or set in progress." role="img"/>
-                            <field name="legend_done" nolabel="1"/>
+                            <div class="o_row" colspan="2">
+                                <label for="legend_normal" string=" " class="o_status oe_project_kanban_legend"
+                                    title="Task in progress. Click to block or set as done."
+                                    aria-label="Task in progress. Click to block or set as done." role="img"/>
+                                <field name="legend_normal" nolabel="1"/>
+                            </div>
+                            <div class="o_row" colspan="2">
+                                <label for="legend_blocked" string=" " class="o_status o_status_red oe_project_kanban_legend"
+                                    title="Task is blocked. Click to unblock or set as done."
+                                    aria-label="Task is blocked. Click to unblock or set as done." role="img"/>
+                                <field name="legend_blocked" nolabel="1"/>
+                            </div>
+                            <div class="o_row" colspan="2">
+                                <label for="legend_done" string=" " class="o_status o_status_green oe_project_kanban_legend"
+                                    title="This step is done. Click to block or set in progress."
+                                    aria-label="This step is done. Click to block or set in progress." role="img"/>
+                                <field name="legend_done" nolabel="1"/>
+                            </div>
```

Comment:
Good job to keep the `oe_project_kanban_legend` class, however, if you look the view in mobile view, then you will see the label is not aligned with the field. 
I'm not an expert in CSS, but I think we can do the same using flexbox to keep the same rendering like this: 
```suggestion
                            <div class="row ml-1" colspan="2">
                                <label for="legend_normal" string=" " class="o_status"
                                    title="Task in progress. Click to block or set as done."
                                    aria-label="Task in progress. Click to block or set as done." role="img"/>
                                <field name="legend_normal" nolabel="1"/>
                            </div>
                            <div class="row ml-1" colspan="2">
                                <label for="legend_blocked" string=" " class="o_status o_status_red"
                                    title="Task is blocked. Click to unblock or set as done."
                                    aria-label="Task is blocked. Click to unblock or set as done." role="img"/>
                                <field name="legend_blocked" nolabel="1"/>
                            </div>
                            <div class="row ml-1" colspan="2">
                                <label for="legend_done" string=" " class="o_status o_status_green"
                                    title="This step is done. Click to block or set in progress."
                                    aria-label="This step is done. Click to block or set in progress." role="img"/>
                                <field name="legend_done" nolabel="1"/>
                            </div>
```

Reply:
@xavierbol Done the changes with several modification. Thanks